apps/extension/src/screens/Dashboard/ChooseLayout/ChooseLayoutCard.tsx

```diff
@@ -0,0 +1,38 @@
+import { Image } from 'react-native';
+import { Text, TextBlack, View } from '@walless/ui';
+
+interface Props {
+	name: string;
+	description: string;
+	love?: number;
+	activeUsers?: number;
```

Comment:
consider put those field under an interface named e.g: `interface WalletLayout {...}`, and then:
```typescript
import type { WalletLayout } from 'url-to-types'

interface Props {
  item: WalletLayout;
}
```

It is most likely `WalletLayout` will be defined via GraphQL api (or other kind of api interface), client-side/front-end use generated Typescript interface from the api.